src/Trip/Manipulator.test.ts

```diff
@@ -0,0 +1,287 @@
+import * as chai from 'chai';
+import * as chaiAsPromised from 'chai-as-promised';
+import * as cloneDeep from 'lodash.clonedeep';
+import * as sinon from 'sinon';
+import { SinonSandbox } from 'sinon';
+
+import { Place } from '../Places';
+import { setEnvironment } from '../Settings';
+import * as PlaceExpectedResults from '../TestData/PlacesExpectedResults';
+import * as TripExpectedResults from '../TestData/TripExpectedResults';
+import * as Manipuator from './Manipulator';
+import { Day, ItineraryItem, Trip } from './Trip';
+
+let sandbox: SinonSandbox;
+chai.use(chaiAsPromised);
+
+describe('TripManipulator', () => {
+	before((done) => {
```

Comment:
Are those `before, after, beforeEach ` really needed for this test? It is synchronous function so I think 

let sandbox: SinonSandbox;
 +chai.use(chaiAsPromised);
and other stuff is not necessary